modules/apps/segments/segments-service/src/main/java/com/liferay/segments/service/impl/SegmentsExperienceLocalServiceImpl.java

```diff
@@ -320,6 +320,21 @@ public void deleteSegmentsExperiences(
 		}
 	}
 
+	@Override
+	public long fetchDefaultSegmentsExperienceId(long plid) {
+		Layout layout = _layoutLocalService.fetchLayout(plid);
+
+		SegmentsExperience segmentsExperience = fetchSegmentsExperience(
```

Comment:
Hey @ealonso !!!

i think that layout could be null, and maybe you can obtain a NPE when you call layout.getGroupId(). What do you think?

Thanks!!